split.go

```diff
@@ -0,0 +1,119 @@
+package girc
+
+import (
+	"bytes"
+	"strconv"
+	"unicode/utf8"
+)
+
+// splitFunc is the type used for functions implementing splitting of
+// too long IRC messages. The function is passed an event which must be
+// split into multiple and an event length which shall not be exceeded.
+type splitFunc func(event *Event, maxLen int) []*Event
+
+var splitFuncs = map[string]splitFunc{
+	PRIVMSG: splitPRIVMSG,
+}
+
+// getIntOption returns the integer value for a given IRC server option
+// name. If the value is not a valid integer or not available, the
+// given default value is returned instead.
+func getIntOption(client *Client, key string, def int) (val int) {
+	var err error
+	strval, success := client.GetServerOption(key)
```

Comment:
This requires tracking to be enabled. We would either stop splitting messages if tracking is disabled or simply return the default value in this case.